Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift

```diff
@@ -66,8 +134,10 @@ public class SimplifiedLoginViewController: UIViewController {
         return view
     }()
     
-    public init() {
+    public init(viewModel: SimplifiedLoginViewModel) {
```

Comment:
Can we put init methods below class parameters and extract constraints setting into a separate method? I think it will be more readable. What do you think?

Reply:
Sure. Good to have structure.
I updated the views to have the order. Properties, inits, methods.